backend/test/Test/Parser.hs

```diff
@@ -0,0 +1,89 @@
+module Test.Parser
+  ( spec_parser
+  ) where
+
+import Universum
+
+import qualified Data.ByteString.Lazy as L
+import qualified Data.Set as S
+
+import Codec.Xlsx (CellValue(..), Worksheet, ixSheet)
+import Codec.Xlsx.Parser (toXlsx)
+import Test.Hspec (Spec, describe, it, shouldBe, shouldThrow)
+
+import Edna.ExperimentReader.Error (ExperimentParsingError(..))
+import Edna.ExperimentReader.Parser (processWorkSheet)
+import Edna.ExperimentReader.Types (CellType(..), PointYX(..))
+import Edna.Web.Types (ExperimentalMeasurement(..))
+
+tablesPath :: FilePath
+tablesPath = "./resources/testSheets"
+
+getWorkSheet :: FilePath -> Text -> IO Worksheet
+getWorkSheet filePath sheetName = do
+  file <- L.readFile filePath
+  maybe (throwM WorksheetNotFound) pure $ (toXlsx file) ^? ixSheet sheetName
+
+tableBaseTest :: FilePath -> Text -> [Text] -> [Text] -> Int -> Int -> IO ()
+tableBaseTest path sheetName eCompounds eTargets eOutliers eNumberOfMeasurements = do
+  workSheet <- getWorkSheet (tablesPath <> path) sheetName
```

Comment:
1. You can use `</>` from `System.FilePath` here and avoid placing `/` in front of each `path` that you pass here.
2. Ideally you should avoid all `/` in this file because one day someone may want to run these tests on Windows for whatever reason. It's a bad style to hardcode `/` as path delimiter. 

I can accept usage of `/` (and discourage it) because it's unlikely that anybody will run it on Windows (but in theory Haskell works fine on Windows and for example someone can develop there), but let's at least insert `/` here rather than outside of this function.

Reply:
Oh, interesting, I actually don't know and couldn't quickly find the answer. Probably the best way to know it is to test it on Windows ourselves… I would not bother about that for now, just make sure that paths are platform-independent in `.hs` files.